roles/weave/tasks/main.yml

```diff
@@ -29,21 +23,29 @@
   tags:
     - weave
 
+# Create weave bridge.
 - name: bring up weave bridge
   command: ifup weave
   sudo: yes
+
+- name: upload weave template service
```

Comment:
this gets uploaded here (will the service get started, as part of some notify handler?) might be better to jig the order so this is before the docker start (incase there is no notify handler)??